### ISSUE TITLE
Improve method display

### ISSUE THREAD

**Issue by xlc** [polkadot-js/extension#417]
What I want

<img width="701" alt="Screen Shot 2020-09-01 at 10 31 08 PM" src="https://user-images.githubusercontent.com/1201310/91839227-d6c03800-eca2-11ea-8070-72a1c555b9e2.png">

What I got

<img width="424" alt="Screen Shot 2020-09-01 at 10 30 14 PM" src="https://user-images.githubusercontent.com/1201310/91839156-b4c6b580-eca2-11ea-8a1a-c85f2652d657.png">

I don't think average user is able to figure out what is call index 0x0f03

**Comment by jacogr** [comment]
Yes, I know, but really not going to pull params decoding as in apps in here. Maybe someday that package gets a bit of TLC, is extract and can be used. Until then, have to live with `toHuman` for a while longer.

**Comment by jacogr** [comment]
As to specifics, the `callIndex` is not very "human", may want to remove it 

https://github.com/polkadot-js/api/blob/master/packages/types/src/generic/Call.ts#L232

**Comment by xlc** [comment]
Another thing will be helpful is make arguments named. So instead of array, make it
```
{
"proposal": "0x...",
"index": 13,
"approve": true,
}
```
